Code/Framework/AzCore/AzCore/Serialization/Json/RegistrationContext.h

```diff
@@ -69,7 +69,13 @@ namespace AZ
                 return HandlesTypeId(azrtti_typeid<T>(), overwriteExisting);
             }
 
+#if defined(AZ_COMPILER_MSVC)
+            // There is a bug with the MSVC compiler when using the 'auto' keyword here. It appears that MSVC is unable to distinguish between a template
+            // template argument with a type variadic pack vs a template plate argument with a non-type auto variadic pack. 
```

Comment:
repeat: typo with "template plate"

```suggestion
            // template argument with a type variadic pack vs a template template argument with a non-type auto variadic pack. 
```